couch/tests/test_unit.py

```diff
@@ -33,15 +33,15 @@ def test_config(test_case, extra_config, expected_http_kwargs):
 
         check.check(instance)
 
-        http_wargs = dict(
```

Comment:
Personally I actually find the `dict` syntax easier to read (fewer strings), I'm surprised that ruff has an opinion about it.